packages/components/src/components/contract-card/contract-card-items/lookbacks-card-body.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { getDisplayStatus, getCardLabels, isCryptocurrency, CONTRACT_TYPES } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import ContractCardItem from './contract-card-item';
+import MobileWrapper from '../../mobile-wrapper';
+import Money from '../../money';
+import Text from '../../text';
+import ArrowIndicator from '../../arrow-indicator';
+import { ResultStatusIcon } from '../result-overlay/result-overlay';
+import { TGeneralContractCardBodyProps } from './contract-update-form';
+
+type TLookBacksCardBody = Pick<TGeneralContractCardBodyProps, 'contract_info' | 'currency' | 'is_sold'> & {
+    progress_slider_mobile_el: React.ReactNode;
+    indicative?: number | null;
+};
+
+const LookBacksCardBody = ({
+    contract_info,
+    currency,
+    is_sold,
+    indicative,
+    progress_slider_mobile_el,
+}: TLookBacksCardBody) => {
+    const { buy_price, contract_type, sell_price, profit, multiplier } = contract_info;
+    const { INDICATIVE_PRICE, MULTIPLIER, PROFIT_LOSS, POTENTIAL_PROFIT_LOSS, PAYOUT, PURCHASE_PRICE } =
+        getCardLabels();
+
+    const getPayoutLimit = (contract_type?: string, multiplier?: number) => {
```

Comment:
Do you think it can be a good practice to move the getPayoutLimit function outside the component. Defining getPayoutLimit outside of the component prevents it from being re-created on every render

Reply:
The problem here is `<Localize />`. When it's outside the component it might not update the translation if the user change language

Reply:
I get it , Thanks.